.github/workflows/pr.yml

```diff
@@ -0,0 +1,69 @@
+name: PR
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Can we run this workflow  on master as well to catch semantic merge conflicts? And maybe releases as well just to be safe.

Reply:
We could. But the repo settings just allow rebase commits so the HEAD of your PR branch represents the HEAD of the main branch, if the PR lands.

Reply:
TBH even with the checks I'd feel way better if the tests ran on `main` and tags.

Reply:
Well, I'd say, only `main`. Because when we tag a release, we do that on a main branch that passed already, right?